test/common/ssl/context_impl_test.cc

```diff
@@ -279,24 +269,21 @@ TEST_F(SslContextImplTest, TestGetCertInformationWithExpiration) {
 TEST_F(SslContextImplTest, TestNoCert) {
   Json::ObjectSharedPtr loader = TestEnvironment::jsonLoadFromString("{}");
   ClientContextConfigImpl cfg(*loader, factory_context_);
-  ContextManagerImpl manager(time_system_);
-  Stats::IsolatedStoreImpl store;
-  ClientContextSharedPtr context(manager.createSslClientContext(store, cfg));
+  ClientContextSharedPtr context(manager_.createSslClientContext(store_, cfg));
   EXPECT_EQ(nullptr, context->getCaCertInformation());
   EXPECT_EQ(nullptr, context->getCertChainInformation());
 }
 
 class SslServerContextImplTicketTest : public SslContextImplTest {
 public:
-  static void loadConfig(ServerContextConfigImpl& cfg, Event::TestTimeSystem& time_system) {
-    ContextManagerImpl manager(time_system);
-    Stats::IsolatedStoreImpl store;
+  static void loadConfig(ServerContextConfigImpl& cfg, ContextManagerImpl& manager,
```

Comment:
why not make this non-static so manager and store can be ref'd from class?

Reply:
Yes. Agree. We can change it like that.